### PR TITLE
Implement 1-3,1-4,1-5,minor fixes to the other cases

### PR DESCRIPTION
**Pull request by mhagen-vmware**
New tests, some cleanup on old tests and fixed some out of date doc.

Passed build:
https://e2e-ci.vmware.run/vmware/vic/154

### CODE REVIEW

File: tests/resources/Util.robot [hunk 1]
```diff
@@ -28,6 +28,11 @@ Install VIC Appliance To Test Server
     ${ret}=  Run Keyword If  "${certs}" == "true"  Remove String  ${ret}  info
     Run Keyword If  "${certs}" == "true"  Set Suite Variable  ${params}  ${ret}
     Log To Console  Installer completed successfully: ${vch-name}...
+    # Required due to #1109
+    Sleep  10 seconds
```
Comment (mdubya66) [inline tests/resources/Util.robot]: should we add a #fixme?

File: tests/test-cases/Group1-Docker-Commands/1-2-Docker-Pull.robot [hunk 2]
```diff
@@ -72,15 +72,15 @@ Pull non-existent image
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} pull fakebadimage
     Log  ${output}
     Should Be Equal As Integers  ${rc}  1
-    # Github issue #757
+    ${status}=  Get State Of Github Issue  757
```
Comment (mdubya66) [inline tests/test-cases/Group1-Docker-Commands/1-2-Docker-Pull.robot]: #fixme?

**Comment by mdubya66** [comment]
Mostly nits. lgtm

**Comment by matthewavery** [comment]
LGTM

**Comment by jzt** [comment]
LGTM

**Pull request merged**